app/globals.css

```diff
@@ -30,3 +31,10 @@ button, input[type="submit"], input[type="reset"] {
 	cursor: pointer;
 	outline: inherit;
 }
+
+input {
+	color: inherit;
+	border: none;
+	font: inherit;
+	outline: inherit;
```

Comment:
Indention is different from others, and `tab` is used here, which is different from others